tests/test_envs.py

```diff
@@ -33,15 +44,20 @@ class TestEnvs:
 
     def test_seed(self, env: gym.Env, env_name: str):
         """Tests environment seeding."""
-        envs.test_seed(env, env_name, DETERMINISTIC_ENVS)
+        envs.test_seed(env, env_name, DETERMINISTIC_ENVS, ATARI_NO_FRAMESKIP_ENVS)
 
     def test_premature_step(self, env: gym.Env):
         """Tests if step() before reset() raises error."""
         envs.test_premature_step(env, skip_fn=pytest.skip, raises_fn=pytest.raises)
 
-    def test_rollout_schema(self, env: gym.Env):
+    # if env is atari then don't wait until done else do normal thing
+    # or maybe force done=True?
+    def test_rollout_schema(self, env: gym.Env, env_name: str):
         """Tests if environments have correct types on `step()` and `reset()`."""
-        envs.test_rollout_schema(env)
+        if env_name in ATARI_ENVS:
```

Comment:
Why?

Reply:
Atari environments have very long episode lengths once we apply the AutoResetWrapper (~100k frames), such that the tests would take too long to run if we had to wait for every environment to finish running.

Reply:
I guess if we really want to check that `env.step` gets the right return types after the end of the episode for atari environments, we could save rom states close to the end of an episode for each environment and load them in this test. My guess is that that's too annoying to be worth it given that taking steps after `done` is true is "an abuse of the Gym API", but I could be wrong.

Reply:
Will add comments saying why we treat atari differently.